recipe/meta.yaml

```diff
@@ -10,16 +10,16 @@ source:
     sha256: 36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d
 
 build:
-    number: 1
+    number: 3
```

Comment:
With conda >= 4.1.2, I think it should already be here.

On Tue, Jun 28, 2016, 07:12 Filipe <notifications@github.com> wrote:

> In recipe/meta.yaml
> <https://github.com/conda-forge/zlib-feedstock/pull/6#discussion_r68744397>
> :
>
> > @@ -10,16 +10,16 @@ source:
> >      sha256: 36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d
> >
> >  build:
> > -    number: 1
> > +    number: 3
>
> I cannot wait for that 😄
>
> —
> You are receiving this because you are subscribed to this thread.
>
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/zlib-feedstock/pull/6/files/2bb7d0a03612958ac44536a11bd54a367f01ef72#r68744397>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACV-a5UTysIohpKsmX2LVgvVVyaCS9Kks5qQQ-mgaJpZM4IkkTX>
> .
>